frontend/app/src/components/account-management/Login.vue

```diff
@@ -316,6 +317,19 @@ export default class Login extends Vue {
 
   mounted() {
     this.loadSettings();
+    if (this.username) {
+      this.focusPassword();
+    }
+  }
+
+  private focusPassword() {
+    this.$nextTick(() =>
+      setTimeout(() => {
+        (this.$refs.password as any).$el
+          .querySelector('input:not([type=hidden])')
+          .focus();
+      }, 100)
```

Comment:
This was a bit more complicated than I expected. I had to go through a couple of so threads.